### ISSUE TITLE
No led bars on the side, vertical cant be set to 0

### ISSUE THREAD

**Issue by BertusWillem** [fabsenet/adrilight#107]
This is a hobby project only, everyone is doing this in his/her free time so be nice!

Try to describe your problem as detailed as possible. Provide screenshots of your settings and photos of your hardware setup.

- Make sure rebooting does not fix your issue!
- can you see the demo effect?
yes
- can you see the preview?
yes

I can't put the vertical lights to 0, I don't have led bars on the side.
How can this be forced?

**Comment by fabsenet** [comment]
Do you have leds at the top **and** the bottom?

I am curios, can you explain the reasoning behind leaving the sides out?

**Comment by BertusWillem** [comment]
Yes i only want the ambient light coming form the bottom and the top. If i leave the vertical on 1, 1 of the leds is getting the color of the sides. That's why i want to change it to 0, but i cant do it in the software. 
Is there a way i can change it in the background?

**Comment by fabsenet** [comment]
I think, it should be possible to change adrilight to allow a zero there but requires some testing if that breaks something. I will see, if I can change this, but my time is currently rather limited.

**Comment by BertusWillem** [comment]
Thanks, i am very curious!

**Comment by BertusWillem** [comment]
@fabsenet did you have time to correct this issue?
